<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editUser</class>
 <widget class="QDialog" name="editUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="GlobalStyleSheet" native="true">
     <property name="styleSheet">
      <string notr="true">/**************************QLineEdit***************************/

QLineEdit
{
	background-color: rgba(0, 0, 0, 0);
	border: None;
	border-bottom: 2px solid rgba(105, 118, 132, 255);
	color: rgba(255, 255, 255, 230);
	padding-bottom: 7px;
}

QLineEdit:focus {
	border-bottom: 2px solid rgba(151, 175, 201, 255);
}

QLineEdit:disables {
	color: rgba(150, 150, 150, 230);
}

/**************************QLabel***************************/
QLabel
{
	color: rgba(250, 250, 250, 140)
}

/**************************QComboBox***************************/

QComboBox
{
	border:2px solid rgba(105, 118, 132, 255);
	background-color: transparent;
	border-radius: 6px;
	padding-left: 15px;
	font-size: 16px;
}

QComboBox:enabled{
	color: rgba(250, 250, 250, 140);
}

QComboBox:disabled
{
	border: 2px solid rgba(61, 69, 77, 255);
	color: rgba(120, 120, 120, 140);
}

QComboBox::down-arrow:disabled
{   
	image: url(:/resources/Icons/down_icon_gray.png);
	width: 12px;
    height: 12px;
}

QComboBox::down-arrow
{   
	image: url(:/resources/Icons/down_icon_gray.png);
	width: 12px;
    height: 12px;
}

QComboBox::drop-down {
    subcontrol-origin: padding; 
    subcontrol-position: bottom right;
    width: 30px; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px; 
}

QComboBox::item:selected {
    border-left: 3px solid rgba(151, 175, 201, 255);
	background-color: rgba(10, 14, 26, 255);
}

QComboBox QAbstractItemView {
	background-color: rgba(10, 14, 26, 255);
	border-radius: 6px;
}

QComboBox:focus{
	border-bottom: 2px solid rgba(151, 175, 201, 255);
}

/**************************QTableWidget***************************/

QTableWidget {
    background-color: transparent;
	gridline-color: #3C4D61;
	color: rgb(220, 220, 220);
	border: None;
}

QHeaderView::section {
    background-color: #01011A;
    padding: 5px;
	border: None;
	border-bottom: 2px solid #3C4D61;
	/*border-left: 1px solid #3C4D61;*/
	font-weight: bold;
	color: rgb(220, 220, 220);
	min-height: 40px;
}

QHeaderView::section:first {
   border-left: None;
}

QTableWidget::item{
	border-bottom: 1px solid #3C4D61;
}

QTableWidget::item:selected {
    background-color: #01011A;
	color: rgb(220, 220, 220);
}



/**************************QScrollBar***************************/

QScrollBar:vertical {
    border: none;
    background: #01011A;
    width: 7px;
    border-radius: 3px;
}

QScrollBar::handle:vertical {
    background: #3C4D61;
    min-height: 20px;
    border-radius: 3px;
}

QScrollBar::add-line:vertical {
    border: none;
    background: #01011A;
    height: 20px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    border: none;
    background: #01011A;
    height: 20px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    border: none;
    width: 3px;
    height: 3px;
    background: transparent;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

QScrollBar:horizontal {
    border: none;
    background: #01011A;
    height: 7px;
    border-radius: 3px;
}

QScrollBar::handle:horizontal {
    background: #3C4D61;
    min-width: 20px;
    border-radius: 3px;
}

QScrollBar::add-line:horizontal {
    border: none;
    background: #01011A;
    width: 20px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    border: none;
    background: #01011A;
    width: 20px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
    border: none;
    width: 3px;
    height: 3px;
    background: transparent;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="LocalStyleSheet" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="PointStyleSheet" native="true">
           <property name="styleSheet">
            <string notr="true">/**************************PmainFrameStyle***************************/

*[styleSheet=&quot;PmainFrameStyle&quot;]
{

	
background-color:rgb(65, 59, 71);
	border-radius: 20px;
}

/**************************PinnerFrameStyle***************************/

*[styleSheet=&quot;PinnerFrameStyle&quot;]
{
	background-color: transparent;
	border-radius: 20px;
}

/**************************PtopFrameStyle***************************/

*[styleSheet=&quot;PtopFrameStyle&quot;]
{
	background-color: transparent;
	border: None;
}


/**************************PpagesFrameStyle***************************/

*[styleSheet=&quot;PpagesFrameStyle&quot;]
{
	background-color: transparent;
	border-radius: 15px;
}


/***********************PtempPushButtonStyle************************/

*[styleSheet=&quot;PtempPushButtonStyle&quot;]
{
	border: 0px;
	background-color: rgba(0,0,0,0);
}

*[styleSheet=&quot;PtempPushButtonStyle&quot;]:pressed{
	border: 0px;
	background-color: rgba(0,0,0,0);
}

/***********************PeyePushButtonStyle************************/

*[styleSheet=&quot;PeyePushButtonStyle&quot;]
{
	border: 0px;
	background-color: rgba(0,0,0,0);
}

*[styleSheet=&quot;PeyePushButtonStyle&quot;]:pressed
{
	border: 0px;
	background-color: rgba(0,0,0,0);
}

/****************PpagesUnderLinePushButtonStyle*****************/

*[styleSheet=&quot;PpagesUnderLinePushButtonStyle&quot;]
{
	color: rgba(250, 250, 250, 140);
	border: 0px;
	background-color: rgba(0,0,0,0);
}

*[styleSheet=&quot;PpagesUnderLinePushButtonStyle&quot;]:hover
{
	text-decoration: underline;
}

*[styleSheet=&quot;PpagesUnderLinePushButtonStyle&quot;]:pressed
{
	color: rgba(250, 250, 250, 140);
	border: 0px;
	background-color: rgba(0,0,0,0);
	text-decoration: underline;
}

/****************PpagesMainPushButtonStyle*****************/

*[styleSheet=&quot;PpagesMainPushButtonStyle&quot;]
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255, 255, 255, 210);
	border-radius: 5px;
}

*[styleSheet=&quot;PpagesMainPushButtonStyle&quot;]:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}

*[styleSheet=&quot;PpagesMainPushButtonStyle&quot;]:pressed
{
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(105, 118, 132, 200);
}

/**************************PmessageLabelStyle***************************/

*[styleSheet=&quot;PmessageLabelStyle&quot;]
{
	color:rgb(255, 99, 94);
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_29">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="main_frame">
              <property name="styleSheet">
               <string notr="true">PmainFrameStyle</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="inner_frame">
                 <property name="styleSheet">
                  <string notr="true">PinnerFrameStyle</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>9</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="top_frame">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">PtopFrameStyle</string>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item alignment="Qt::AlignRight|Qt::AlignTop">
                      <widget class="QPushButton" name="close_btn">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">#close_btn{
	border: 0px;
	background-color: rgba(0,0,0,0);
}

#forgot_btn:pressed{
	border: 0px;
	background-color: rgba(0,0,0,0);
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="resources/resource.qrc">
                         <normaloff>:/resources/Icons/close.png</normaloff>:/resources/Icons/close.png</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="page_frame">
                    <property name="styleSheet">
                     <string notr="true">PpagesFrameStyle</string>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>12</number>
                     </property>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_28">
                       <property name="spacing">
                        <number>18</number>
                       </property>
                       <property name="leftMargin">
                        <number>20</number>
                       </property>
                       <property name="rightMargin">
                        <number>20</number>
                       </property>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_14">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QLineEdit" name="edituser_username_lineEdit">
                           <property name="minimumSize">
                            <size>
                             <width>0</width>
                             <height>0</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>16777215</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="font">
                            <font>
                             <pointsize>11</pointsize>
                            </font>
                           </property>
                           <property name="placeholderText">
                            <string>Username</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="edituser_temp_btn">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                           <property name="minimumSize">
                            <size>
                             <width>20</width>
                             <height>16</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>20</width>
                             <height>16</height>
                            </size>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">PtempPushButtonStyle</string>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_16">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QLineEdit" name="edituser_password_lineEdit">
                           <property name="minimumSize">
                            <size>
                             <width>0</width>
                             <height>0</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>16777215</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="font">
                            <font>
                             <pointsize>11</pointsize>
                            </font>
                           </property>
                           <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                           </property>
                           <property name="placeholderText">
                            <string>Password</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="edituser_password_eye_btn">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                           <property name="cursor">
                            <cursorShape>PointingHandCursor</cursorShape>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">PeyePushButtonStyle</string>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                           <property name="icon">
                            <iconset resource="resources/resource.qrc">
                             <normaloff>:/resources/Icons/eye.png</normaloff>:/resources/Icons/eye.png</iconset>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_18">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QLineEdit" name="edituser_confirm_password_lineEdit">
                           <property name="minimumSize">
                            <size>
                             <width>0</width>
                             <height>0</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>16777215</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="font">
                            <font>
                             <pointsize>11</pointsize>
                            </font>
                           </property>
                           <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                           </property>
                           <property name="placeholderText">
                            <string>Confirm Password</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="edituser_confirm_password_eye_btn">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                           <property name="cursor">
                            <cursorShape>PointingHandCursor</cursorShape>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">PeyePushButtonStyle</string>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                           <property name="icon">
                            <iconset resource="resources/resource.qrc">
                             <normaloff>:/resources/Icons/eye.png</normaloff>:/resources/Icons/eye.png</iconset>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <widget class="QComboBox" name="edituser_role_comboBox">
                         <property name="enabled">
                          <bool>true</bool>
                         </property>
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>35</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>290</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <item>
                          <property name="text">
                           <string>Role</string>
                          </property>
                         </item>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignHCenter">
                        <widget class="QPushButton" name="apply_btn">
                         <property name="minimumSize">
                          <size>
                           <width>150</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>150</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <pointsize>12</pointsize>
                           <weight>75</weight>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="cursor">
                          <cursorShape>PointingHandCursor</cursorShape>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">PpagesMainPushButtonStyle</string>
                         </property>
                         <property name="text">
                          <string>APPLY</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="edit_user_message_label">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">PmessageLabelStyle</string>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resource.qrc"/>
  <include location="resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
